KNOWN_ISSUES.md

```diff
@@ -120,10 +120,12 @@ consult the Cloudant documentation for further information.
 
 ### Session authentication
 
-Session authentication does not work with compressed request bodies when the server is CouchDB <= 3.1.1 or Cloudant <= 8169. Disabling gzip compression for requests is required when using session authentication with these server versions:
+Session authentication does not work with compressed request bodies when the server is CouchDB <= 3.1.1 or Cloudant <= 8169. [Disabling gzip compression](#disabling-request-body-compression) for requests is required when using session authentication with these server versions
+
+### Disabling request body compression
```

Comment:
Optional task: it would be good to move this part also into the template.

Reply:
Considering every language has this chapter. It would be better to move this file also to template doc in sdks.

Reply:
Yes every SDK repo will have this section but the code block will be specific to the language.  I think it's easier to maintain if we have the code in each repo rather than try and handle multiple code blocks in the sdks repo.

Reply:
Okay.